src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,80 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  onPageChange: (number: number) => void;
+  currentPage: number;
+  perPage: number;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  onPageChange: handleChangePage,
+  currentPage,
+  perPage,
+}) => {
+  const totalPageSelectors = Math.ceil(total / perPage);
+  const pageSelectors = getNumbers(1, totalPageSelectors);
+
+  const isFirstPage = currentPage === 1;
+  const isLastPage = currentPage === totalPageSelectors;
+
+  const isPrevEnabled = currentPage > 1;
+  const isNextEnabled = !isLastPage;
+
+  return (
+    <ul className="pagination">
+      <li
+        className={classNames('page-item',
+          {
+            disabled: !isPrevEnabled,
+          })}
+      >
```

Comment:
```suggestion
  const isPrevDisabled = currentPage === 1;
  const isNextDisabled = isLastPage;

  return (
    <ul className="pagination">
      <li
        className={classNames('page-item',
          {
            disabled: isPrevDisabled,// no need for double negation :)
            // moreover, you could just dump the additional variables
            // since logic is pretty straight
            disabled: currentPage === 1,
            // same with nextPage button
          })}
      >
```